app/models/app.js

```diff
@@ -17,6 +17,7 @@ export default DS.Model.extend(ProvisionableMixin, {
   lastDeployOperation: DS.belongsTo('operation', {async:true}),
   currentImage: DS.belongsTo('image', {async:true}),
 
+  reloadWhileProvisioning: true,
```

Comment:
That is true - the only statuses in use for an app are `pending` and `provisioned`, as well as 'deprovisioning'. 'pending' doesn't actually reload the model - only 'provisioning' and 'deprovisioning' do. 

I'll override `shouldReload()` (and refactor it a bit) since it will be a bit clearer on what's actually going on.